samples/nrf9160/nrf_cloud_rest_device_message/README.rst

```diff
@@ -0,0 +1,110 @@
+.. _nrf_cloud_rest_device_message:
+
+nRF9160: nRF Cloud REST Device Message
+######################################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The REST Device Message sample demonstrates how to use the `nRF Cloud REST API`_ to send a `Device Message <nRF Cloud Device Messages_>`_ using the ``SendDeviceMessage`` endpoint. The d2c topic is used in non-bulk mode.
+
+Requirements
+************
+
+The sample supports the following development kits:
+
+.. table-from-rows:: /includes/sample_board_rows.txt
+   :header: heading
+   :rows: nrf9160dk_nrf9160_ns
+
+The sample requires an `nRF Cloud`_ account.
+
+.. note::
+   This sample requires modem firmware v1.3.x or later.
+
+Overview
+********
+
+Pressing the button sends a button press message that you can view in the Terminal card of the `nRF Cloud`_ portal (check the Device-to-Cloud channel).
+The device must already be provisioned.
+If your device is not yet provisioned, you can do so over REST by enabling the :kconfig:`CONFIG_REST_DEVICE_MESSAGE_DO_JITP` option and following the instructions printed to UART upon boot.
+
+Limitations
+***********
+
+This sample requires the network carrier to provide date and time to the modem.
+Without a valid date and time, the modem cannot generate JWTs with an expiration time.
+Note that REST connections are transient, unlike MQTT connections which are usually long-lived.
+
+User interface
+**************
+
```

Comment:
You might want to consider describing the buttons and LEDs in this section like in https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/samples/openthread/coap_client/README.html